### ISSUE TITLE
Module has incorrect dependency on "terraform-aws-ecs-fargate"

### ISSUE THREAD

**Issue by sergeykuzmich** [cn-terraform/terraform-aws-sonarqube#2]
It loads module "terraform-aws-ecs-fargate" version 2.0.1 which has an issue with `ipc_mode`
https://github.com/cn-terraform/terraform-aws-ecs-fargate/issues/2

**Comment by jnonino** [comment]
Hi, how are you?
Can you try with the new version? [2.0.4](https://registry.terraform.io/modules/cn-terraform/sonarqube/aws/2.0.4)

**Comment by sergeykuzmich** [comment]
Now there is another issue
```
Error: Incorrect attribute value type

  on .terraform/modules/sonarqube.ecs_fargate.ecs-fargate-service/cn-terraform-terraform-aws-ecs-fargate-service-a50b265/main.tf line 230, in resource "aws_lb" "lb":
 230:   subnets                          = var.public_subnets
    |----------------
    | var.public_subnets is list of list of string with 1 element
```

**Comment by jnonino** [comment]
Should be fixed now in version [2.0.5](https://registry.terraform.io/modules/cn-terraform/sonarqube/aws/2.0.5)

**Comment by sergeykuzmich** [comment]
Seems working!

Thank you a lot!
